rpc/openapi/openapi.yaml

```diff
@@ -51,10 +51,10 @@ info:
 
         ws ws://localhost:26657/websocket
         > { "jsonrpc": "2.0", "method": "subscribe", "params": ["tm.event='NewBlock'"], "id": 1 }
-  version: "Master"
+  version: "v0.34"
```

Comment:
The v0.34 tag points to main?

Reply:
No, this was incorrectly labelled. Take a look at <https://docs.tendermint.com/v0.34/rpc/>: the "Master" tag to the top-right of the "Tendermint RPC" heading should actually read "v0.34", because it's the OpenAPI spec for v0.34.

This is mostly a cosmetic thing, but I think it's important we label things correctly.